src/main/kotlin/no/nav/syfo/persistering/api/SendRegisterPapirSykmeldingOppgaveRest.kt

```diff
@@ -23,19 +24,15 @@ import no.nav.syfo.client.findBestSamhandlerPraksis
 import no.nav.syfo.clients.KafkaProducers
 import no.nav.syfo.log
 import no.nav.syfo.model.ReceivedSykmelding
-import no.nav.syfo.model.SmRegisteringManuellt
+import no.nav.syfo.model.SmRegisteringManuell
 import no.nav.syfo.model.Status
-import no.nav.syfo.persistering.handleManuellOppgave
+import no.nav.syfo.pdl.service.PdlPersonService
 import no.nav.syfo.persistering.handleOKOppgave
 import no.nav.syfo.service.ManuellOppgaveService
 import no.nav.syfo.service.mapsmRegisteringManuelltTilFellesformat
 import no.nav.syfo.service.toSykmelding
-import no.nav.syfo.util.LoggingMeta
-import no.nav.syfo.util.extractHelseOpplysningerArbeidsuforhet
-import no.nav.syfo.util.fellesformatMarshaller
-import no.nav.syfo.util.get
-import no.nav.syfo.util.getAccessTokenFromAuthHeader
-import no.nav.syfo.util.toString
+import no.nav.syfo.util.*
```

Comment:
Normalt bruker vi ikke stjerneimports, så her vil det bli mye diff hvis vi ikke har de samme settingene i intellij :) For meg er det egentlig det samme, så lenge vi er enige om hva vi skal bruke Vi kan gjerne ta diskusjonen i backend-kanalen :)

Reply:
Tror det er noe litt rart med IntelliJ-oppsettet mitt, for jeg får ikke stjerneimports til å funke skikkelig. Jeg har satt i settings at den ikke skal bruke stjerneimports med mindre det er 999 importeringer fra samme pakke, og fjernet alle standardimports, men allikevel så insisterer den på å bruke stjerneimports.

Den klarer heller ikke lese markdown-filer (som Readme.MD), så kanskje dette er dagen hvor jeg setter opp IntelliJ på nytt :p

Reply:
Jøss, så rart! Kanskje verdt å sette opp på nytt da ja :D